code/modules/medical/genetics/bioEffects/useless.dm

```diff
@@ -288,6 +293,7 @@
 	name = "Bee Production"
 	desc = "Encourages growth of bees in the subject's body."
 	id = "drunk_bee"
+	icon_state = "unknown"
```

Comment:
This is the behavior on the base bio effect so you do not need to specify here. Ditto for the other `"unknown"` ones added.

Reply:
Ah, I see! I'll edit that shortly, ty! :D